Update note about using atomic blocks with firewall CSP (#11526)
MicrosoftDocs/windows-itpro-docs: windows/client-management/mdm/firewall-csp.md

> [!NOTE]
> Firewall rules in the FirewallRules section must be wrapped in an Atomic block in SyncML, either individually or collectively.
>

> > Atomic blocks are "all or nothing", if a firewall rule or firewall setting in an Atomic block fails to be applied, the entire Atomic block will fail to be applied.

Reviewer: ```suggestion
> Atomic blocks are "all or nothing", if a firewall rule or firewall setting in an Atomic block fails to be applied, the entire Atomic block fails to be applied.
```

>

> > If an Atomic block contains a firewall rule or firewall setting that is not supported on a particular Windows OS version, the entire Atomic block will fail to be applied on that Windows version. For example, firewall rules with IcmpTypesAndCodes are only supported on Windows 11, applying an Atomic block that contains a rule with IcmpTypesAndCodes on Windows 10 will fail.

Reviewer: ```suggestion
> If an Atomic block contains a firewall rule or firewall setting that is not supported on a particular Windows OS version, the entire Atomic block fails to be applied on that Windows version. For example, firewall rules with IcmpTypesAndCodes are only supported on Windows 11, applying an Atomic block that contains a rule with IcmpTypesAndCodes on Windows 10 fails.
```

> [!NOTE]
> Firewall rules in the FirewallRules section must be wrapped in an Atomic block in SyncML, either individually or collectively.

> > Atomic blocks are "all or nothing", if a firewall rule or firewall setting in an Atomic block fails to be applied, the entire Atomic block will fail to be applied.
> > If an Atomic block contains a firewall rule or firewall setting that is not supported on a particular Windows OS version, the entire Atomic block will fail to be applied on that Windows version. For example, firewall rules with IcmpTypesAndCodes are only supported on Windows 11, applying an Atomic block that contains a rule with IcmpTypesAndCodes on Windows 10 will fail.

Reviewer 2: ```suggestion
>
> Atomic blocks are "all or nothing", if a firewall rule or firewall setting in an Atomic block fails to be applied, the entire Atomic block will fail to be applied.
>
> If an Atomic block contains a firewall rule or firewall setting that is not supported on a particular Windows OS version, the entire Atomic block will fail to be applied on that Windows version. For example, firewall rules with IcmpTypesAndCodes are only supported on Windows 11, applying an Atomic block that contains a rule with IcmpTypesAndCodes on Windows 10 will fail.
```
